test/TestProjects/TestAppWithContents/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "Microsoft.NETCore.Runtime": "1.0.1-beta-*",
```

Comment:
I am leaving these test project.jsons as such just to have some test diversity. `dotnet new` and `dotnet publish` in the E2E test must cover StdLib scenario.